File: Programming Basics/Programming Basics - JavaScript/Exercises/04. Complex Conditions/07. Fruit Shop/FruitShop.js
function fruitShop([arg1, arg2, arg3]) {
    let fruit = arg1.toLowerCase();
    let day = arg2.toLowerCase();
    let quantity = Number(arg3);

    let price;
    if (day == "monday" || day == "tuesday" || day == "wednesday"
                        || day == "thursday" || day == "friday") {
        if (fruit == "banana") {
            price = 2.50 * quantity;
        }
        else if (fruit == "apple") {
            price = 1.20 * quantity;
        }
        else if (fruit == "orange") {
            price = 0.85 * quantity;
        }
        else if (fruit == "grapefruit") {
            price = 1.45 * quantity;
        }
        else if (fruit == "kiwi") {
            price = 2.70 * quantity;
        }
        else if (fruit == "pineapple") {
            price = 5.50 * quantity;
        }
        else if (fruit == "grapes") {
            price = 3.85 * quantity;
        }
        else {
            console.log("error")
        }
    }

    else if (day == "saturday" || day == "sunday") {
        if (fruit == "banana") {
            price = 2.70 * quantity;
        }
        else if (fruit == "apple") {
            price = 1.25 * quantity;
        }
        else if (fruit == "orange") {
            price = 0.90 * quantity;
        }
        else if (fruit == "grapefruit") {
            price = 1.60 * quantity;
        }
        else if (fruit == "kiwi") {
            price = 3.00 * quantity;
        }
        else if (fruit == "pineapple") {
            price = 5.60 * quantity;
        }
        else if (fruit == "grapes") {
            price = 4.20 * quantity;
        }
        else {
            console.log("error")
        }
    }
    else {
        console.log("error")
    }

    console.log(price);
}

/* Uncomment to test:
fruitShop(["apple", "tuesday", "2"]);
fruitShop(["orange", "sunday", "3"]);
fruitShop(["kiwi", "monday", "2.5"]);
    */

File: Programming Basics/Programming Basics - C#/Exercises/04. Complex Conditionals/04. Complex Conditionals/Fruit Shop/Fruit Shop.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Fruit_Shop
{
    class Program
    {
        static void Main(string[] args)
        {
            input:
            var fruit = Console.ReadLine().ToLower();
            var day = Console.ReadLine().ToLower();
            var quantity = double.Parse(Console.ReadLine());


            if (fruit == "banana")
            {
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(2.50 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(2.70 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else if (fruit == "apple")
            {
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(1.20 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(1.25 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else if (fruit == "orange")
            {
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(0.85 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(0.90 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else if (fruit == "grapefuit")
            { 
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(1.45 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(1.50 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else if (fruit == "kiwi")
            { 
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(2.70 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(3.00 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else if (fruit == "pineapple")
            {
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(5.50 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(5.60 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else if (fruit == "grapes")
            {
                if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday")
                {
                    Console.WriteLine(Math.Round(3.85 * quantity, 2));
                }

                else if (day == "Saturday" || day == "Sunday")
                {
                    Console.WriteLine(Math.Round(4.20 * quantity, 2));
                }

                else
                {
                    Console.WriteLine("error");
                }
            }

            else
            {
                Console.WriteLine("error");
            }

            goto input;
        }
    }
}
